commune.lic

```diff
@@ -0,0 +1,412 @@
+	    @devotion_level = 14
+      when 'After a moment, you see a vision of your god who calls to you by name, "Come here, my child'
+	    @devotion_level = 13
+      when 'After a moment, you see a silent vision of your god'
+	    @devotion_level = 12
+      when 'After a moment, you sense a slight pressure on your shoulder'
+	    @devotion_level = 11
+      when 'After a moment, you see a vision of your god, though the visage is cloudy'
+	    @devotion_level = 10
+      when 'After a moment, you sense that your god is pleased with your devotion'
+	    @devotion_level = 9
+      when 'After a moment, you sense that your god knows your name'
+	    @devotion_level = 8
+      when 'After a moment, you sense that your god is aware of your devotion'
+	    @devotion_level = 7
+      when 'After a moment, you sense a distinct link between you and your god'
+	    @devotion_level = 6
+      when 'After a moment, you sense that your efforts have not gone unnoticed'
+	    @devotion_level = 5
+      when 'After a moment, you sense that your god is barely aware of you'
+	    @devotion_level = 4
+      when 'You call out to your god, but there is no answer'
+	    @devotion_level = 3
+      when 'You close your eyes and start to concentrate'
+	    @devotion_level = 2
+      when 'You feel unclean and unworthy'
+	    @devotion_level = 1
+	  when 'You sense nothing special from your communing'
+	    @devotion_level = 0
+	  end
+	  return @devotion_level
+	end
+
+  ##################### CARVING ########################
+  
+  def create_bead(bead_type='meraud')
+    symbols = { 'kertigen' => 'raven', 'divyaush' => 'welkin', 'zachriedek' => 'raccoon',
+	            'hodierna' => 'unicorn', 'berengaria' => 'cow', 'asketi' => 'adder',
+	            'meraud' => 'wolf', 'firulf' => 'owl', 'kerenhappuch' => 'shrew',
+	            'damaris' => 'panther', 'phelim' => 'nightingale', 'dergati' => 'shrike',
+	            'everild' => 'boar', 'kuniyo' => 'wolverine', 'trothfang' => 'centaur',
+	            'truffenyi' => 'ox', 'alamhif' => 'magpie', 'huldah' => 'weasel',
+	            "hav'roth" => 'cobra', "peri'el" => 'kingsnake', 'ushnish' => 'viper',
+	            'eluned' => 'dolphin', 'lemicus' => 'albatross', 'drogor' => 'shark',
+	            'glythtide' => 'ram', 'saemaus' => 'donkey', "be'ort" => 'coyote',
+	            'tamsine' => 'cat', 'albreda' => 'dove', 'harawep' => 'spider',
+	            'faenella' => 'wren', 'murrula' => 'phoenix', 'idon' => 'heron',
+	            'chadatru' => 'lion', 'rutilor' => 'mongoose', 'botolf' => 'goshawk',
+	            "urrem'tier" => 'scorpion', 'eylhaar' => 'jackal', 'aldauth' => 'vulture'
+		      }
+	if symbols.key?("#{bead_type}")
+	  @shape = symbols["#{bead_type}"]
+	elsif symbols.value?("#{bead_type}")
+	  @shape = bead_type
+	else
+	  missing_item('a valid carving target')
+	end
+	echo "Carving: #{@shape}"
+    if GameObj.right_hand.noun == 'block'
```

Comment:
Yes but it could carve a block, then tap a block, then carve a bead. The extra check just skips the tapping if the block already exists in the right hand. It was added so you can ;commune carve and then ;commune truffenyi without tapping or moving items. The action ;commune carve is intended to leave a prayer bead in your right hand.